pkg/controllers/work/apply_controller.go

```diff
@@ -629,6 +672,144 @@ func buildManifestAppliedCondition(err error, action applyAction, observedGenera
 	}
 }
 
+func (r *ApplyWorkReconciler) createConfigMap(ctx context.Context, configMapName string, owner metav1.OwnerReference, manifestHash, lastModifiedConfig string) error {
+	configMap := &v1.ConfigMap{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      configMapName,
+			Namespace: fleetSystemNamespace,
+		},
+		Data: map[string]string{},
+	}
+	configMap.Data[manifestHashKey] = manifestHash
+	configMap.Data[lastAppliedConfigKey] = lastModifiedConfig
+	// add applied work owner reference. let's check if it works
+	addOwnerRef(owner, configMap)
+	if err := r.spokeClient.Create(ctx, configMap); err != nil {
+		klog.ErrorS(err, "config map create failed", "configMap")
```

Comment:
It looks as though are going to get multiple error logs that contain the same message. 
See  line 716. https://github.com/Azure/fleet/pull/343/files#diff-9c51763423a3a9a026f47bc21555619edb62a1cf676c669b643e57b53c39c7caR716